src/commands/php_list.rs

```diff
@@ -0,0 +1,95 @@
+use clap::App;
+use clap::SubCommand;
+use std::env;
+use std::path::Path;
+use std::path::PathBuf;
+use std::str;
+use glob::glob;
+use std::process::Command;
+use std::process::Stdio;
+use regex::Regex;
+
+pub(crate) fn command_config<'a, 'b>() -> App<'a, 'b> {
+    SubCommand::with_name("php:list").about("List all available PHP executables")
+}
+
+pub(crate) fn php_list() {
+    let path_string = env::var_os("PATH").unwrap();
+    let path_dirs = path_string
+        .to_str()
+        .unwrap()
+        .split(get_path_separator().as_str())
```

Comment:
Below, I made a suggestion that update `get_path_separator` to return a `&'static str` instead of a `String`, so that the following suggestion is possible:

```suggestion
        .split(get_path_separator())
```